includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -1233,6 +1237,10 @@ public function generate_create_intent_request( $order, $prepared_source ) {
 			$request['statement_descriptor'] = $full_request['statement_descriptor'];
 		}
 
+		if ( isset( $full_request['statement_descriptor_suffix'] ) ) {
+			$request['statement_descriptor_suffix'] = $full_request['statement_descriptor_suffix'];
```

Comment:
Is this here so that the param can be injected via a filter? Do we know how this would interact with the (concatenated) full descriptor we are passing?

Reply:
Thanks for raising this. This is actually from the very beginning when I was testing out the prefix + suffix case. As explained above, it won't work so I removed it in a8780a4.